lib/clouds/vcd_cloud_ops.py

```diff
@@ -12,842 +13,85 @@
 
 '''
     Created on Jan 7, 2013
-
     vCloud Director Object Operations Library
-
     @author: Daniel R. Bowers
 '''
-from time import time, sleep
-
-from socket import gethostbyname
 
 from lib.auxiliary.code_instrumentation import trace, cbdebug, cberr, cbwarn, cbinfo, cbcrit
-from lib.auxiliary.data_ops import str2dic, DataOpsException
-from lib.remote.network_functions import Nethashget
-
-from shared_functions import CldOpsException, CommonCloudFunctions 
-
-from libcloud.compute.types import Provider
-from libcloud.compute.providers import get_driver
-from libcloud.compute.types import NodeState
+from libcloud_common import LibcloudCmds
```

Comment:
My apologies, it was clearly my fault, when attempting to reconcile a conflict from a previous merge.